### DISCUSSION TITLE
I can´t download 0.22.1 on Windows

### DISCUSSION THREAD

**Discussion by Ultraschorsch** [simulot/immich-go#519] [Q&A]
I want to finally transfer all my fotos etc. from Google to my Immich installation with immich-go. But I can´t download the latest stable version because there seems to be a false positive with 0.22.1. I tried on different networks, browsers and computers but the download gets blocked every time. I can download older versions just fine but I think there has been some changes to Immich that need the most recent stable version to work. Is this a known issue?

**Comment by simulot** [comment]
I'm surprised to that. 
I just checked it on virus total:
https://www.virustotal.com/gui/file/30108d4aa8b64b4f27d7b40eeb7cdc5999c15b1ec2f5a8fc16883fa49c192afc
Nothing significant.

I'm starting a windows VM to test a on fresh install

**Comment by Ultraschorsch** [comment]
I was surprised, too. I also checked with Virustotal and there was nothing special there. I tried downloading the zip with my phone and upload it to Google drive but I also couldn´t download it to my laptop from there.

**Reply by simulot** [reply]
Nothing special on my windows 11 vm freshly updated.
